### PR TITLE
#4 create banner leo component

### LINKED ISSUE #4: Create Banner Leo Component

### PR DESCRIPTION
**Pull request by ghost**
Close #4

### CODE REVIEW

File: src/components/Price/Price.vue [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    &__value {
+      grid-area: value;
+    }
+
+    &--inline {
+      grid-template-columns: 1fr;
+      grid-template-areas: "text before value";
+    }
+  }
+</style>
```
Comment (javipuche) [inline src/components/Price/Price.vue]: Missing more custom properties

File: src/components/Price/PriceButton.vue [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                default: false
+            },
+            buttonIcon: {
+                type: [String, Boolean],
+                default: undefined
+            },
+            buttonText: {
+                type: String,
+                default: undefined
+            }
```
Comment (javipuche) [inline src/components/Price/PriceButton.vue]: Pass properties as an object, as in the banner component

**Review by javipuche — CHANGES_REQUESTED** [review]
Review several things

File: src/components/Price/Price.vue [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        :class="{'c-price--inline' : inline}"
+    >
+        <div class="c-price__text">
+            {{text}}
+        </div>
+        <div class="c-price__before">
+            {{before}}{{currency}}
+        </div>
+        <div class="c-price__value">
+            {{value}}{{currency}}
```
Comment (jjlmoya) [inline src/components/Price/Price.vue:13]: In a close future we need to improve this doing a service for a better i18n, in some currency it's displayed {{currency}}{{value}}

Issue opened #30

**Review by jjlmoya — APPROVED** [review]

File: src/components/Banner/BannerSirius.vue [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        shadow
+        radius
+        overlay
+        :picture="picture"
+    >
+        <AtIconList />
+        <AtPriceButton
+            text="Desde"
+            before="600"
+            value="900"
```
Comment (javipuche) [inline src/components/Banner/BannerSirius.vue]: Bind an object

File: src/components/Price/PriceButton.vue [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+                :text="text"
+                :before="before"
+                :value="value"
+            />
+        </div>
+        <div class="c-price-button__button">
+            <Button
+                :icon="buttonIcon"
+                :text="buttonText"
+            />
```
Comment (javipuche) [inline src/components/Price/PriceButton.vue]: Bind an object

**Review by javipuche — CHANGES_REQUESTED** [review]
Pass properties as an object

**Review by javipuche — APPROVED** [review]

**Pull request merged**